buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -94,14 +92,34 @@ def construct_failure_info(failed_jobs_per_flag):
         if jobs:
             github_url = INCOMPATIBLE_FLAGS[flag]
             info_text.append(f"* **{flag}** " + get_html_link_text(":github:", github_url))
-            for job in jobs.values():
-                link_text = get_html_link_text(job["name"], job["web_url"])
-                info_text.append(f"  - {link_text}")
+            info_text += merge_and_format_jobs(jobs.values(), "  - ")
     if len(info_text) == 1:
         return None
     return info_text
 
 
+def merge_and_format_jobs(jobs, line_prefix):
+    # Merges all jobs for a single pipeline into one line.
+    # Example:
+    #   pipeline (platform1)
+    #   pipeline (platform2)
+    #   pipeline (platform3)
+    # with line_prefix ">> " becomes
+    #   >> pipeline: platform1, platform2, platform3
+    jobs_per_pipeline = collections.defaultdict(list)
+    for job in jobs:
+        pipeline, platform = get_pipeline_and_platform(job)
+        jobs_per_pipeline[pipeline].append(get_html_link_text(platform, job["web_url"]))
+
+    return ["{}{}: {}".format(line_prefix, pipeline, ", ".join(platforms)) for pipeline, platforms in jobs_per_pipeline.items()]
```

Comment:
Can we make the pipeline also bold, `{}{}: {}` -> `{}**{}**: {}`, maybe that will make the pipeline name more attractive.

Reply:
Done